res/css/views/messages/_ViewSourceEvent.scss

```diff
@@ -35,13 +35,15 @@ limitations under the License.
         mask-size: auto 12px;
         visibility: hidden;
         background-color: $accent;
-        mask-image: url('$(res)/img/feather-customised/maximise.svg');
+        mask-image: url("$(res)/img/element-icons/maximise-expand.svg");
     }
 
     &.mx_ViewSourceEvent_expanded .mx_ViewSourceEvent_toggle {
         mask-position: 0 bottom;
         margin-bottom: 7px;
-        mask-image: url('$(res)/img/feather-customised/minimise.svg');
+        width: 10px;
+        height: 10px;
```

Comment:
These don't seem correct to me... The height is forcing the icon to appear at the top of the expanded event instead of the bottom, and the width seems too small. I believe they can just be removed, but please test yourself as well.

To test this feature, enable "Show hidden events in timeline" in labs and look for a hidden event like a reaction. It will have an expand / collapse icon. When toggling between expand and collapse, the icon should always be under your cursor (bottom right of the event when expanded).